src/LanguageServer.Engine/CompletionProviders/ItemMetadataCompletionProvider.cs

```diff
@@ -221,7 +221,7 @@ IEnumerable<CompletionItem> GetElementCompletions(XmlLocation location, ProjectD
             HandleTriggerCharacters(triggerCharacters, projectDocument, ref targetRange);
 
             // These items are handled by PackageReferenceCompletion.
-            if (itemType == "PackageReference" || itemType == "DotNetCliToolReference")
+            if (itemType == "PackageReference" || itemType == "PackageVersion" || itemType == "DotNetCliToolReference")
```

Comment:
```suggestion
            if (itemType is "PackageReference" or "PackageVersion" or "DotNetCliToolReference")
```

Reply:
Isn't the `is` operator normally for type-testing? I get that you can use it with the new pattern matching features but is it really clearer than `==`?